_data/authors.yml

```diff
@@ -15,3 +15,8 @@ ahmddp:
   twitter: AhmedDemirpolat
   github: ahmddp
   about: Senior Software Developer
+
+emrahkaya:
+  display_name: Emrah Kaya
+  github: emrahkaya
+  about: Sense a soul in search of answers
```

Comment:
http://orcz.com/Diablo_3:_I_Sense_a_Soul_in_Search_of_Answers_Achievement ?? :)

Reply:
Yeah, sure :) . But in fact, I'm more a fan of Starcraft. https://vignette.wikia.nocookie.net/starcraft/images/9/9c/ProtossArbiterPissed00SC1.ogg/revision/latest?cb=20170508192522